src/ripple/app/rdb/backend/RWDBDatabase.h

```diff
@@ -463,11 +457,9 @@ class RWDBDatabase : public SQLiteDatabase
         for (const auto& [_, accountData] : accountTxMap_)
         {
             size += sizeof(AccountID) + sizeof(AccountTxData);
-            size += accountData.transactions.size() * sizeof(AccountTx);
-            for (const auto& [_, innerMap] : accountData.ledgerTxMap)
+            for (const auto& [_, txVector] : accountData.ledgerTxMap)
             {
-                size += sizeof(uint32_t) +
-                    innerMap.size() * (sizeof(uint32_t) + sizeof(size_t));
+                size += sizeof(uint32_t) + txVector.size() * sizeof(AccountTx);
```

Comment:
is this right?
`    using AccountTx =
        std::pair<std::shared_ptr<Transaction>, std::shared_ptr<TxMeta>>;`
shouldn't we iterate txVector and find the byte sizes of the txns and meta?